resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -13,6 +14,7 @@ if [ -r "$restartfile" ];  then
         echo "Service not running so cannot be reloaded" 1>&2
         exit 1
     fi
+    sleep 0.1
     cur=$(head -n 1 "$restartfile")
     initial=$cur
```

Comment:
I switched the order for grabbing the `initial` value to before sending the HUP signal in c46b39da.